demo/runners/support/agent.py

```diff
@@ -1504,14 +1504,20 @@ async def get_invite(
                 invi_params = {
                     "auto_accept": json.dumps(auto_accept),
                 }
-                payload = {"mediation_id": self.mediator_request_id}
+                payload = {
+                    "mediation_id": self.mediator_request_id,
+                    "handshake_protocols": ["https://didcomm.org/connections/1.1"],
```

Comment:
Sorry for the dumb question, but should this still be connections?

Reply:
Nice catch. I meant to change this to `did-exchange/1.1`. Not exactly sure why this worked.